app/Http/Controllers/User/UserController.php

```diff
@@ -85,12 +86,25 @@ public function update(Request $request)
     /**
      * Removing a user from the system.
      *
-     * @param  int  $id
-     * @return \Illuminate\Http\Response
+     * @param Request $request
+     * @return \Illuminate\Http\JsonResponse|\Symfony\Component\HttpFoundation\Response
      */
-//    public function destroy($id)
-//    {
-//
-//    }
+    public function destroy(Request $request)
+    {
+        $user = $request->user();
+
+        if ($user->transfersOut()->where('status', 'pending')->count()) {
+            return $this->buildFailedValidationResponse($request, 'You must complete all transfers.');
+        }
+
```

Comment:
Does this include pending transactions that others have sent you AND transactions that you have sent other users.

What I mean is, what if someone sends you money, deletes there account, and then you decline the transaction. What happens to that money?